build/root/WORKSPACE

```diff
@@ -91,7 +91,7 @@ docker_pull(
     digest = "sha256:1a05a58432254268c31ef5c8d9c21f3d01a40611b14707de6ac2772c0793bd13",
```

Comment:
you need to update this, or else it's not going to have any effect on bazel builds.

Reply:
update to `sha256:fc1b461367730660ac5a40c1eb2d1b23221829acf8a892981c12361383b3742b` and then this lgtm

Reply:
Done!